src/context/metrics.ts

```diff
@@ -92,7 +92,12 @@ export class MetricsRegistry {
 			'signed_token_total',
 			'Number of issued signed private tokens.'
 		);
-		this.r2RequestsTotal = this.create('counter', 'r2_requests_total', 'Number of accesses to R2');
+		this.r2RequestsDurationMs = this.create(
+			'histogram',
+			'r2_requests_duration_ms',
```

Comment:
nit: the naming here is inconsistent with the `request_time_total` metric above, even though they're both histograms of durations.

Reply:
the above should be updated. It's a typo. I'll add a dedicated issue, as it's for another work.